src/blocks/Caddy121-2/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Intro</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link href="style.css" rel="stylesheet" />
+  </head>
+
+  <body>
+    <div id="intro">
+      <h1>
+        <span>I'm</span>
+        <div class="caddy">Caddy121</div>
+      </h1>
+    </div>
+    <script src="script.js"></script>
```

Comment:
You can remove the Javascript file if you don't need it. No need to commit an empty file.

Reply:
Got it, thank you!